pkg/controllers/edgeconnect/secret/secret.go

```diff
@@ -80,3 +92,17 @@ func safeEdgeConnectCfg(cfg config.EdgeConnect) string {
 
 	return string(safe)
 }
+
+func addDefaultServiceAccount() (config.Secret, error) {
```

Comment:
It's not necessarily the default service account. It's the token of whatever service account is configured.
```suggestion
func createServiceAccountTokenSecret() (config.Secret, error) {
```

Reply:
fixed in 50f5f58f